rosdep/base.yaml

```diff
@@ -1811,6 +1811,16 @@ libhdf5-dev:
   debian: [libhdf5-dev]
   fedora: [hdf5-devel]
   ubuntu: [libhdf5-dev]
+libi2c-dev:
+  arch:
+    pacman:
+      packages: [linux-api-headers]
+  debian:
+    apt:
+      packages: [libi2c-dev]
+  ubuntu:
+    apt:
+      packages: [libi2c-dev]
```

Comment:
I know I saw people doing it without that stuff for arch in other places, I can get rid of it.  I mostly figured it was better to be more explicit